examples/cpp/custom_payload_pool/Application.hpp

```diff
@@ -53,4 +53,4 @@ class Application
 } // namespace fastdds
 } // namespace eprosima
 
-#endif /* _FASTDDS_CUSTOM_PAYLOAD_POOL_APPLICATION_HPP_ */
+#endif /* FASTDDS_EXAMPLES_CPP_CUSTOM_PAYLOAD_POOL__APPLICATION_HPP */
```

Comment:
NIT unify comment macro style, we agree with `//`, apply to the rest of the headers
```suggestion
#endif // FASTDDS_EXAMPLES_CPP_CUSTOM_PAYLOAD_POOL__APPLICATION_HPP
```